src/elements/ngm-height-slider.ts

```diff
@@ -70,7 +70,7 @@ export class NgmHeightSlider extends LitElementI18n {
     const altitude = this.viewer.scene.globe.getHeight(camera.positionCartographic) || 0;
     let height = Math.round(valueToHeight(Number(input.value)));
     const snapDistance = 150; // snap to 0 from 150m/-150m
```

Comment:
Can you also change the snapDistance to f.e. 800? (currently I nearly don't feel that it snaps, I'd expect that it would snap as soon as the icon approaches the limit. This is at about +/-1000 meters, but it depends of course on the screen size. Can we try 800m therefore?)

Reply:
updated